packages/dappmanager/src/modules/https-portal/migration.ts

```diff
@@ -138,3 +144,36 @@ export function migrateCoreNetworkAndAliasInCompose(
 
   compose.write();
 }
+
+function isComposeNetworkAndAliasMigrated(
+  composeNetwork: ComposeNetwork | null,
+  serviceNetwork: ComposeServiceNetwork | null,
+  composeVersion: string,
+  alias: string
+): boolean {
+  // semver can only work with version of 3 numbers. e.g: 3.5.0
+  const composeVersionCleaned = semver.clean(composeVersion + ".0", {
+    loose: true
+  });
+  const minimumComposeVersionCleaned = semver.clean(
+    params.MINIMUM_COMPOSE_VERSION + ".0",
+    {
+      loose: true
+    }
+  );
```

Comment:
Consider adding a function parseComposeVersion() to de-duplicate the logic and explain why is this necessary